### PR TITLE
Update local-groups.mdx

### PR DESCRIPTION
**Pull request by Dadsrcworkbench**
I am the admin for the San Diego mesh so I added the link to our discord because we are always to looking to expand the mesh.

### CODE REVIEW

File: docs/community/local-groups.mdx [hunk 1]
```diff
@@ -26,6 +26,8 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 - [Laguna Mesh](https://lagunamesh.com)
 - [Mission Viejo Mesh](https://missionviejomesh.org/)
 - [Bay Area Mesh](https://canvis.app/z2k8a8)
+- [San Diego Mesh]
+(https://discord.gg/HzE248x2)
```
Comment (tekstrand) [inline docs/community/local-groups.mdx:30]: This needs to be on the same line as `[San Diego Mesh]` with no space between

**Review by tekstrand — CHANGES_REQUESTED** [review]

**Comment by pdxlocations** [comment]
@Dadsrcworkbench could you please sign the CLA?

**Comment by Dadsrcworkbench** [comment]
I signed the CLASent from my iPhoneOn Feb 3, 2024, at 12:24 PM, pdxlocations ***@***.***> wrote:﻿
@Dadsrcworkbench could you please sign the CLA?

**Review by pdxlocations — APPROVED** [review]

File: docs/community/local-groups.mdx (hunk 1 above)
Comment (pdxlocations) [reply docs/community/local-groups.mdx]: I pushed a commit to fix

**Pull request merged**